scalatest-test/src/test/scala/org/scalatest/AssertionsSpec.scala

```diff
@@ -71,8 +71,8 @@ class AssertionsSpec extends AnyFunSpec {
       }
     }
     it("should compare nulls in a satisfying manner") {
-      val n1: String = null
-      val n2: String = null
+      val n1: String | Null = null
+      val n2: String | Null = null
```

Comment:
**Nullable variable**
Variables that really can be null and should therefore have a `| Null` type.